This patch sets the socket group to sudo instead of wheel
diff --git a/lact-daemon/src/config.rs b/lact-daemon/src/config.rs
index c90f203..145dab2 100644
--- a/lact-daemon/src/config.rs
+++ b/lact-daemon/src/config.rs
@@ -18,7 +18,7 @@ use tokio::{sync::mpsc, time};
 use tracing::{debug, error};
 
 const FILE_NAME: &str = "config.yaml";
-const DEFAULT_ADMIN_GROUPS: [&str; 2] = ["wheel", "sudo"];
+const DEFAULT_ADMIN_GROUPS: [&str; 2] = ["sudo", "wheel"];
 /// Minimum amount of time between separate config reloads
 const CONFIG_RELOAD_INTERVAL_MILLIS: u64 = 50;
 
